File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int get_height(void);
void print_hills(int height);

int main(void)
{

        int h = get_height();

    print_hills(h);

    return 0;
 }

// Get input from user
int get_height(void)
{
    int h;
    do
    {
        h = get_int("How high should we build, sir?: ");
    }
    while (h < 1 || h > 8);

    return h;
}

// Make the two hills
void print_hills(int h)
{
    for (int i = 0; i < h; i++)
    {
        for (int j = 0; j < h; j++)
        {
            if (i + j < h - 1)
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }

        printf("  ");

        for (int k = 0; k < i + 1; k++)
        {
            printf("#");
        }

        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
# Runs main
def main():
    height = get_height()
    print_hills(height)


# Gets input between 1-8
def get_height():
    while True:
        try:
            h = int(input("How high should we build, sir?: "))
            if 1 <= h <= 8:
                return h
            # Loops until a valid number is input
            else:
                print("Input number between 1 and 8.")
        except ValueError:
            print("Please enter a valid number.")


# Prins "#" or " " depending on the value
def print_hills(h):
    for i in range(h):
        print(" " * (h - i - 1) + "#" * (i + 1) + "  " + "#" * (i + 1))


if __name__ == "__main__":
    main()
